### PR TITLE
Mbriskar windup 606

### PR DESCRIPTION
**Pull request by jsight**
Continuation of https://github.com/windup/windup/pull/569

### CODE REVIEW

**Comment by windupgithubbot1** [comment]
Refer to this link for build results (access rights to CI server needed): 
http://10.3.8.5//job/windup-pr-builder/787/
Test PASSed.

**Comment by windupgithubbot1** [comment]
Refer to this link for build results (access rights to CI server needed): 
http://10.3.8.5//job/windup-core-PR-dependents-check/154/
Test PASSed.

**Comment by mbriskar** [comment]
Oh I didn't realize we don't have the handler :) 
Thanks

File: config/api/src/main/java/org/jboss/windup/config/metadata/AbstractRulesetMetadata.java [hunk 1]
```diff
@@ -33,6 +33,12 @@ public String getID()
     }
 
     @Override
+    public String getDescription()
+    {
+        return null;
```
Comment (lincolnthree) [inline config/api/src/main/java/org/jboss/windup/config/metadata/AbstractRulesetMetadata.java]: This needs to be implemented?

Comment (lincolnthree) [inline config/api/src/main/java/org/jboss/windup/config/metadata/AbstractRulesetMetadata.java]: If this is intended to be a default, it should be "" empty string but not null IMO.

Comment (jsight) [inline config/api/src/main/java/org/jboss/windup/config/metadata/AbstractRulesetMetadata.java]: Good point... will fix.

**Comment by windupgithubbot1** [comment]
Refer to this link for build results (access rights to CI server needed): 
http://10.3.8.5//job/windup-pr-builder/788/<h2>Failed Tests: <span class='status-failure'>1</span></h2><h3><a name='org.jboss.windup.tests:windup-tests' /><a href='http://10.3.8.5/job/windup-pr-builder/788/org.jboss.windup.tests$windup-tests/testReport'>org.jboss.windup.tests:windup-tests</a>: <span class='status-failure'>1</span></h3><ul><li><a href='http://10.3.8.5/job/windup-pr-builder/788/org.jboss.windup.tests$windup-tests/testReport/org.jboss.windup.tests.application/WindupArchitectureSpringSmallTest/testRunWindupSmallSpringApp/'><strong>org.jboss.windup.tests.application.WindupArchitectureSpringSmallTest.testRunWindupSmallSpringApp</strong></a></li></ul>
Test FAILed.

**Comment by windupgithubbot1** [comment]
Refer to this link for build results (access rights to CI server needed): 
http://10.3.8.5//job/windup-core-PR-dependents-check/155/<h2>Failed Tests: <span class='status-failure'>2</span></h2><h3><a name='org.jboss.windup.quickstarts:windup-ejb-beanutils-async-rules-java' /><a href='http://10.3.8.5/job/windup-core-PR-dependents-check/155/org.jboss.windup.quickstarts$windup-ejb-beanutils-async-rules-java/testReport'>org.jboss.windup.quickstarts:windup-ejb-beanutils-async-rules-java</a>: <span class='status-failure'>1</span></h3><ul><li><a href='http://10.3.8.5/job/windup-core-PR-dependents-check/155/org.jboss.windup.quickstarts$windup-ejb-beanutils-async-rules-java/testReport/org.jboss.windup.qs.rules/EjbBeanUtilsAsyncUsageRuleProviderTest/testRules/'><strong>org.jboss.windup.qs.rules.EjbBeanUtilsAsyncUsageRuleProviderTest.testRules</strong></a></li></ul><h3><a name='org.jboss.windup.quickstarts:windup-weblogic-javaee-servlet-rules-java' /><a href='http://10.3.8.5/job/windup-core-PR-dependents-check/155/org.jboss.windup.quickstarts$windup-weblogic-javaee-servlet-rules-java/testReport'>org.jboss.windup.quickstarts:windup-weblogic-javaee-servlet-rules-java</a>: <span class='status-failure'>1</span></h3><ul><li><a href='http://10.3.8.5/job/windup-core-PR-dependents-check/155/org.jboss.windup.quickstarts$windup-weblogic-javaee-servlet-rules-java/testReport/org.jboss.windup.qs.rules/WebLogicServletAnnotationTest/testJavaHints/'><strong>org.jboss.windup.qs.rules.WebLogicServletAnnotationTest.testJavaHints</strong></a></li></ul>
Test FAILed.

**Comment by windupgithubbot1** [comment]
Refer to this link for build results (access rights to CI server needed): 
http://10.3.8.5//job/windup-pr-builder/792/<h2>Failed Tests: <span class='status-failure'>1</span></h2><h3><a name='org.jboss.windup.tests:windup-tests' /><a href='http://10.3.8.5/job/windup-pr-builder/792/org.jboss.windup.tests$windup-tests/testReport'>org.jboss.windup.tests:windup-tests</a>: <span class='status-failure'>1</span></h3><ul><li><a href='http://10.3.8.5/job/windup-pr-builder/792/org.jboss.windup.tests$windup-tests/testReport/org.jboss.windup.tests.application/WindupArchitectureSpringSmallTest/testRunWindupSmallSpringApp/'><strong>org.jboss.windup.tests.application.WindupArchitectureSpringSmallTest.testRunWindupSmallSpringApp</strong></a></li></ul>
Test FAILed.

**Comment by windupgithubbot1** [comment]
Refer to this link for build results (access rights to CI server needed): 
http://10.3.8.5//job/windup-core-PR-dependents-check/159/<h2>Failed Tests: <span class='status-failure'>2</span></h2><h3><a name='org.jboss.windup.quickstarts:windup-ejb-beanutils-async-rules-java' /><a href='http://10.3.8.5/job/windup-core-PR-dependents-check/159/org.jboss.windup.quickstarts$windup-ejb-beanutils-async-rules-java/testReport'>org.jboss.windup.quickstarts:windup-ejb-beanutils-async-rules-java</a>: <span class='status-failure'>1</span></h3><ul><li><a href='http://10.3.8.5/job/windup-core-PR-dependents-check/159/org.jboss.windup.quickstarts$windup-ejb-beanutils-async-rules-java/testReport/org.jboss.windup.qs.rules/EjbBeanUtilsAsyncUsageRuleProviderTest/testRules/'><strong>org.jboss.windup.qs.rules.EjbBeanUtilsAsyncUsageRuleProviderTest.testRules</strong></a></li></ul><h3><a name='org.jboss.windup.quickstarts:windup-weblogic-javaee-servlet-rules-java' /><a href='http://10.3.8.5/job/windup-core-PR-dependents-check/159/org.jboss.windup.quickstarts$windup-weblogic-javaee-servlet-rules-java/testReport'>org.jboss.windup.quickstarts:windup-weblogic-javaee-servlet-rules-java</a>: <span class='status-failure'>1</span></h3><ul><li><a href='http://10.3.8.5/job/windup-core-PR-dependents-check/159/org.jboss.windup.quickstarts$windup-weblogic-javaee-servlet-rules-java/testReport/org.jboss.windup.qs.rules/WebLogicServletAnnotationTest/testJavaHints/'><strong>org.jboss.windup.qs.rules.WebLogicServletAnnotationTest.testJavaHints</strong></a></li></ul>
Test FAILed.

File: config-xml/rule-schema.xsd [hunk 2]
```diff
@@ -264,7 +265,7 @@
 							</xs:element>
 							<xs:element name="sourceTechnology" type="technology" maxOccurs="unbounded" minOccurs="0"/>
 							<xs:element name="targetTechnology" type="technology" maxOccurs="unbounded" minOccurs="0"/>
-							<xs:element name="tags">
+							<xs:element name="tags" maxOccurs="1" minOccurs="0">
```
Comment (mbriskar) [inline config-xml/rule-schema.xsd]: tags is not used now. Feel free to yield it to me, I have changed it in another PR already

**Comment by windupgithubbot1** [comment]
Refer to this link for build results (access rights to CI server needed): 
http://10.3.8.5//job/windup-pr-builder/804/
Test PASSed.

**Pull request merged**